python/ray/autoscaler/v2/scheduler.py

```diff
@@ -1612,6 +1612,7 @@ def _enforce_idle_termination(
                 id=str(uuid.uuid4()),
                 instance_id=node.im_instance_id,
                 ray_node_id=node.ray_node_id,
+                instance_type=node.node_type,
```

Comment:
I searched for `instance_type` in this PR, but I haven't found any usage of it. Why do we need this?